### DISCUSSION TITLE
1st time using VSP for mandibular reconstruction with FFF

### DISCUSSION THREAD

**Discussion by SteveMaisi** [lassoan/SlicerBoneReconstructionPlanner#58] [Show and tell]
Hi. I'm glad to share my first ever VSP experience using BRP.

Digital Imaging and Communications in Medicine (DICOM) data uploaded and segmentation using 3D Slicer software 
![Picture2](https://user-images.githubusercontent.com/97390689/151353815-41231b85-567a-42ce-a92a-e75cd857f38b.png)

Surgical margins are set about 1 – 1.5cm away from radiological margin of the lesion 
![Picture3](https://user-images.githubusercontent.com/97390689/151356530-636ad3e6-6d02-4bf7-85e1-4c667488865b.png)

Cutting planes are positioned carefully on the mandible to form the desired curvature of the mandible
![Picture1](https://user-images.githubusercontent.com/97390689/151356543-bde1d0be-f8f0-4130-b141-329a14ae8a53.png)

3D image of the tumour (lateral view)
![Picture4](https://user-images.githubusercontent.com/97390689/151353822-b804aa26-3c69-4bee-ae9c-4e0651aced9b.png)

Resection done virtually
![Picture5](https://user-images.githubusercontent.com/97390689/151353824-8d497bd0-26da-47f0-ae67-7275444cd837.png)

Reconstruction with fibula. In a case of condylar head removal is required, the new condyle should be 5mm below the glenoid fossa
![Picture6](https://user-images.githubusercontent.com/97390689/151353828-94d5a204-ca12-4464-9c69-f93986e6539b.png)

3D image of the tumour (bottom view)
![Picture7](https://user-images.githubusercontent.com/97390689/151353833-493ee82b-ce2b-4f5f-afe5-abf915dd192f.png)

After reconstruction with fibula. It is important to make sure angle of mandible is symmetrical at this point.
![Picture8](https://user-images.githubusercontent.com/97390689/151353836-9aaba81a-4223-4848-9dff-bdffe48788a8.png)

Cutting planes on the mandible are transferred to left fibula. Each fibula segment with its measurement. In this case, grey segments will form the condyle and ramus, pink segment is body of mandible and orange segment is anterior mandible.
![Picture9](https://user-images.githubusercontent.com/97390689/151353838-c896fd28-850d-483e-b81a-449d513000d9.png)

Autogeneration of fibula cutting guide
![Picture10](https://user-images.githubusercontent.com/97390689/151353841-1a91fdf2-e93b-4350-9dca-ba78c63feb1c.png)

Fibula cutting guide in STL format
![Picture11](https://user-images.githubusercontent.com/97390689/151353844-95e46edd-aa51-470e-8657-907aadd22f72.png)

Printed fibula cutting guide with PLA materials
![Picture12](https://user-images.githubusercontent.com/97390689/151353846-f9a3ccd1-4caa-4e62-b557-6f1b6b525ace.png)

Printed reconstructed mandible with fibula in PLA materials
![Picture13](https://user-images.githubusercontent.com/97390689/151353849-cf9f929d-2222-49c7-8e0e-4eb4a2f8506d.png)

3D printer
![Picture14](https://user-images.githubusercontent.com/97390689/151353851-e0d443c8-1c8a-4550-a05c-e143ba1b5423.png)

Fibula cutting guide is fitted and fixed to fibula
![Picture15](https://user-images.githubusercontent.com/97390689/151353854-1916f218-295f-4abd-8fb3-eabc6659a6ab.png)
![Picture16](https://user-images.githubusercontent.com/97390689/151353857-4c0c0502-46a5-4a4a-8a88-d8c28d7cbfc4.png)

Intersegmental bones are removed to allow contouring and fixation. Pedicles still attached at this stage
![Picture17](https://user-images.githubusercontent.com/97390689/151353861-c72c033b-1735-4ea4-9c95-8823ce8f420f.png)

Prebend plates are fixed onto the fibula segments. Noted mandibular contour as per planned.
![Picture18](https://user-images.githubusercontent.com/97390689/151353862-aedfdeb6-dad9-4f73-9811-b52da663b5ab.jpg)

Resection of the tumour. In this case is Brown defect class IVc.
![Picture19](https://user-images.githubusercontent.com/97390689/151353867-65e75cf3-4dd9-4ed4-8a0e-c841d16286f9.jpg)

Neo-mandible fixed to native mandible
![Picture20](https://user-images.githubusercontent.com/97390689/151356367-0c2ddf06-8a6e-4375-b5ff-fa107dcb7498.png)

Pre and post-operative facial appearance 
![Picture21](https://user-images.githubusercontent.com/97390689/151353874-d37402c9-f4ce-465b-96ff-f7eb82a7cf92.png)

Pre and post-operative orthopantomogram
![Picture22](https://user-images.githubusercontent.com/97390689/151353877-f92282af-dfac-48c6-8a28-f144f42fe5ae.png)
![Picture23](https://user-images.githubusercontent.com/97390689/151353878-c43516f7-5246-4eb7-a73e-d26e5305d606.jpg)

**Comment by mauigna06** [comment]
Thank you Steve for logging this case. Please keep doing it if you continue using BRP.

Your post is very complete. The only thing that cames to my mind to ask is: what sterilization method did you use for the PLA printed guide?

Also, did you make the mandible resection guide? Could you attach pictures if you did? Thanks

Do you have any feedback to improve the software? Some feature you would like? or something that you'd want to be easier to use?

Thanks for using the software. 

For everyone reading this: Remember this software is not FDA approved so the doctor's criteria is really important for every case.

**Reply by SteveMaisi** [reply]
We only use antiseptic solutions. in this case is anioxyde 1000. Guides were soaked for 15minutes.
Mandible cutting guide cannot b printed as we have an issue with the 3D printer. we proceed with custom-made acrylic guide.
I'm fully aware of this software is available for implant guide and PSI generation. however, i still need to practice more to get used to it. 
What about a positioning guide? to position the fibula (neo-mandible) to the native mandible?

**Reply by mauigna06** [reply]
> What about a positioning guide? to position the fibula (neo-mandible) to the native mandible?

If I understand correctly may be if you prebent the plates by using the "mandible reconstruction model for 3D printing" you can use that as positioning guide.
Also you could leave fixed the mandible surgical guide after the cut (if you design the bridge properly) so the fibulaPieces fixed on the plate are easier to position I think

**Reply by SteveMaisi** [reply]
Correct. we use prebent plates to position the fibula. Ok, will design a bit the mandibular cutting guide so that can also act as a positioning guide. Great idea! thank you

**Reply by mauigna06** [reply]
> Ok, will design a bit the mandibular cutting guide so that can also act as a positioning guide. Great idea! thank you

The idea was from @mrtig

Mauro
